glib/src/translate.rs

```diff
@@ -277,6 +345,31 @@ impl ToGlib for Ordering {
     }
 }
 
+/// A Rust type `T` for which `Option<T>` translates to the same glib type as T.
+// The copy bound on `Self::GlibType` is requested by clippy by virtue of:
+// https://rust-lang.github.io/rust-clippy/master/index.html#declare_interior_mutable_const
```

Comment:
I don't think this makes sense here, but it also doesn't hurt as all types that implement `ToGlib` / `FromGlib` should be `Copy` anyway? I think you can also just require that in those traits directly.

Reply:
Ok!